src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionIndentationTest.java

```diff
@@ -38,9 +38,9 @@ protected String getCheckName() {
     }
 
     @Test
-    public void testOne() throws Exception {
+    public void testDefault() throws Exception {
         final File fileToProcess =
-                new File(getPath("SuppressionXpathRegressionIndentationTestOne.java"));
+                new File(getPath("InputXpathIndentationTestDefault.java"));
```

Comment:
Remove `Test` from the file names. It seems slightly conflicting to name it an input and a test.

Reply:
Done